llvm/include/llvm/ObjCopy/CommonConfig.h

```diff
@@ -245,6 +245,9 @@ struct CommonConfig {
   // Symbol info specified by --add-symbol option.
   SmallVector<NewSymbolInfo, 0> SymbolsToAdd;
 
+  // Integer options
+  int64_t ChangeSectionLMAValAll = 0;
```

Comment:
Should this option really be in `CommonConfig.h` and not the ELF-specific version? Is it actually applicable to non-ELF formats?

Reply:
Possibly applicable but not implemented by this patch. I left it there for now, and added more explanation in the commit message and added the option to ConfigManager.cpp